pkgs/data/fonts/joypixels/default.nix

```diff
@@ -1,24 +1,95 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, acceptLicense ? false }:
+
+let inherit (stdenv.hostPlatform) system;
+
+    throwSystem = throw "Unsupported system: ${system}";
+
+    systemTag = {
+      x86_64-darwin = "nix-darwin";
+      x86_64-linux = "nix-os";
+    }.${system} or throwSystem;
+
+    capitalized = {
+      x86_64-darwin = systemTag;
+      x86_64-linux = "NixOS";
+    }.${system} or throwSystem;
+
+    fontFile = {
+      x86_64-darwin = "Apple%20Color%20Emoji.ttc";
+      x86_64-linux = "joypixels-android.ttf";
+    }.${system} or throwSystem;
+
+    name = {
+      x86_64-darwin = "joypixels-apple-color-emoji.ttc";
+      x86_64-linux = fontFile;
+    }.${system} or throwSystem;
+
+    ext = {
+      x86_64-darwin = "ttc";
+      x86_64-linux = "ttf";
+    }.${system} or throwSystem;
+
+    joypixels-free-license = {
+      spdxId = "LicenseRef-JoyPixels-Free-6.0";
```

Comment:
SPDX is a [license list](https://spdx.org/licenses/), and this license is not part of it, so it doesn't have any SPDX ID. Same for the other `spdxId`s

Reply:
That's not true. There's a list of standard license identifiers but SPDX is much more than that. My identifiers are to spec afaict.

Reply:
Where to spec means, "LicenseRef-<basically anything goes>" : )
[A reference from the spec.](https://spdx.github.io/spdx-spec/appendix-IV-SPDX-license-expressions/#simple-license-expressions)

Reply:
Ah I didn't know that, thanks for telling me!